Detections/MultipleDataSources/BariumIPIOC.yaml

```diff
@@ -0,0 +1,141 @@
+id: 6ee72a9e-2e54-459c-bc9a-9c09a6502a63 
+name: Known Barium IP 
+description: | 
+  'Identifies a match across various data feeds for IP IOCs related to the Barium activity group.  
+References: https://www.justice.gov/opa/pr/seven-international-cyber-defendants-including-apt41-actors-charged-connection-computer' 
```

Comment:
Is References meant to be part of the description field rather than its own heading?

Reply:
Made it part of description now. Thanks..